mirrord-agent/src/file.rs

```diff
@@ -17,11 +18,21 @@ use tracing::{debug, error};
 
 use crate::{error::AgentError, runtime::get_container_pid, sniffer::DEFAULT_RUNTIME, PeerID};
 
-// TODO: To help with `openat`, instead of `HashMap<_, File>` this should be
-// `HashMap<_, RemoteFile`>, where `RemoteFile` is a struct that holds both the `File` + `PathBuf`.
+/// Unqiue file descriptor generator
+pub(crate) fn get_fd() -> usize {
```

Comment:
Might be clearer to call this generate_fd, get implies you're retrieving an existing value